### DISCUSSION TITLE
Importing Data

### DISCUSSION THREAD

**Discussion by davbay1** [benjaminjonard/koillection#585] [Q&A]
Hi everyone,

It doesn't look like there is an option to import data, am I understanding that correctly?

Ideally, an option to import data via CSV would be great.

**Comment by RaimundasZutautas** [comment]
Hi,
I also cannot find any way to import data.
I want to make backups in case something went wrong.
I see option for export and I want to make backups using that but how I can re-import exported files if I build from scratch?
Is there any way but I just unable to find it? :D

**Reply by benjaminjonard** [reply]
Hello,

For the moment their is no way to import data with Koillection. 
You can have a look at [this issue](https://github.com/benjaminjonard/koillection/issues/839) for more information on how to use the SQL file
